keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -380,6 +595,59 @@ def inpaint(
         decoded = ((decoded + 1) / 2) * 255
         return np.clip(decoded, 0, 255).astype("uint8")
 
+    def tokenize_prompt(self, prompt: str) -> tf.Tensor:
+        """Tokenize a phrase prompt.
+
+        Args:
+            prompt (str): The prompt string to tokenize, must be 77 tokens or shorter.
+
+        Returns:
+            phrase (tf.Tensor): The tokenize prompt.
+        """
+        inputs = self.tokenizer.encode(prompt)
+        if len(inputs) > MAX_PROMPT_LENGTH:
+            raise ValueError(
+                f"Prompt is too long (should be <= {MAX_PROMPT_LENGTH} tokens)"
+            )
+        phrase = inputs + [49407] * (MAX_PROMPT_LENGTH - len(inputs))
+        phrase = tf.convert_to_tensor([phrase], dtype=tf.int32)
+        return phrase
+
+    def create_prompt_weights(
```

Comment:
Where is this used?

Reply:
It is used in the Attention Re-weighting method (check the comments above). It creates a numpy array with weights to scale the attention maps.

**Example**: 

```python
from keras_cv.models.stable_diffusion import StableDiffusion

# Initializae the generator
generator = StableDiffusion()

prompt = "a photo of a pizza with pineapple"
prompt_weights = [("pineapple", 2)]
prompt_weights = generator.create_prompt_weights(prompt, prompt_weights)
```

Reply:
Ideally, I should also change the name of the function, because prompt weights can be associated with a similar idea that has the same name (e.g. `prompt`=a green:0.5 forest) - This could also be a cool and much simpler feature to add :)

I think I will rename the method as `create_attention_weights`.

Reply:
Fixed it, updated the PR comment, and added an issue for weighted prompts #1368